js/addToTask.js

```diff
@@ -71,8 +71,8 @@ TRASH_INPUT.addEventListener(CLICK_EVENT, function (event) {
 UL_LIST.addEventListener(CLICK_EVENT, function (event) {
   const element = event.target;
   const elementJob = element.getAttribute("data-job");
-  //  const elementJob = element.attributes.data-job.value;
-  if (elementJob == DELETE) {
-    removeTodos(element, taskList);
+  if (elementJob == DELETE_TODO) {
+    removeTodo(element, taskList);
+    element.removeEventListener(CLICK_EVENT, event);
```

Comment:
`removeEventListener` takes the same event name and the function reference that was used as the argument for `addEventListener`, second arg `event` is not a function reference.

did we register an event handler on `element` before, if not, adding a remove listener is redundant.